staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -357,7 +357,7 @@ func newInformer(
 			// from oldest to newest
 			for _, d := range obj.(Deltas) {
 				switch d.Type {
-				case Sync, Added, Updated:
+				case Sync, Replaced, Added, Updated:
```

Comment:
do we support external consumers of controller.Config / Process? this PR updates our two internal implementations, but would break all external uses, correct?

Reply:
I don't see anything that references these types (specifically cache.Config or cache.Deltas) outside the package. They probably should have been private types...